actinia_stac_plugin/core/stac_collections.py

```diff
@@ -53,6 +53,11 @@ def StacCollectionsList():
                     stac = stac.decode("utf8").replace("'", '"')
                 except Exception:
                     stac = stac
+                # if response is slow (especially with growing collections),
+                # it might be an option to use pickle to store json in redis
+                json_collection = json.loads(stac)
+                json_collection['id'] = i
+                stac = json.dumps(json_collection)
```

Comment:
Wouldn't it make more sense to leave this step out and append `json_collection` directly so that it might not slow down?

Reply:
It makes sense, but I don't know if there is nay reason to store `stac` as a `sting` instead of a `json`

Reply:
Totally, yes. I overlooked the `json.loads` in the next step... thanks for noticing!